Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
Traceback (most recent call last):
  File "/home/ds6237/cheby_test/train_stu.py", line 95, in <module>
    torch.save(lds.state_dict(), lds_path)
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 849, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 716, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ds6237/.conda/envs/torch-env/lib/python3.12/site-packages/torch/serialization.py", line 687, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory ./models/lds_trained does not exist.
slurmstepd: error: *** JOB 64199667 ON della-j14g3 CANCELLED AT 2025-05-11T01:45:29 DUE TO TIME LIMIT ***
